api/src/main/resources/emails/initial_credits_expiration/content.html

```diff
@@ -29,7 +29,7 @@
 <body>
 <img src="${HEADER_IMG}">
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
-<div>This is a notification that the initial credits offered to your account ${USERNAME} by the All of Us Program have all been used.</div>
```

Comment:
This change and rename is a reversion of recent work.  I think it was accidentally included here.

Reply:
@evrii looks like this was merged without the fix
@NehaBroad we should stop the release